README.md

```diff
@@ -66,6 +66,17 @@ cd scan/wpscan
 bundle install --without test --path vendor/bundle
 </pre>
 
+**Let's encrypt**
+
+Run the letsencrypt-auto file in order for Let's encrypt to install its dependencies. 
+
+<pre>
+cd letsencrypt
+./letsencrypt-auto
```

Comment:
This is actually done every time when running letsencrypt (letsencrypt-auto is a wrapper which checks that dependencies are met). The thing is that initially installing all dependencies takes a long time so I prefer to do it in advance.